### ISSUE TITLE
The utilization rate of GPU is 0 when using Synchronized-BatchNorm

### ISSUE THREAD

**Issue by chauhochow** [CSAILVision/semantic-segmentation-pytorch#95]
when i try to train the model on a 4-card machine, the GPU memory continues to increase and is totally occupied. However, all GPU utilization is 0. It means that the program just occupies the GPU but doesn't compute. It happens after several iterations complete. have you met this problem? It seems to the problem of Synchronized-BatchNorm as [https://github.com/vacancy/Synchronized-BatchNorm-PyTorch/issues/3](url) shows. My docker environment is pytorch0.4.1 plus python3.6.[https://hub.docker.com/r/pytorch/pytorch/](url)

**Comment by Tete-Xiao** [comment]
Sorry we never meet this problem. But please keep us informed if you know more about the bug. We can fix the codes thereafter.
